src/client/Client.js

```diff
@@ -181,7 +181,7 @@ class Client extends BaseClient {
       );
       this.sweepMessageInterval = setInterval(
         this.sweepMessages.bind(this),
-        this.options.messageSweepInterval * 1000,
+        this.options.messageSweepInterval * 1_000,
```

Comment:
Am I the only one not a fan of using separators for such small numbers? I personally start using them from 10k+... @discordjs/the-big-3 thoughts?

Reply:
Personally I don’t think it looks very good on 1000 but other 4-digit numbers bigger than that are fine imo. I just ended up changing all because some places had 1000 and other numbers so it would seem odd for one to have it and not the others

Reply:
My thoughts are the same.